src/motion/position.ts

```diff
@@ -275,11 +276,11 @@ export class Position extends vscode.Position {
         word: word,
       };
 
-      if (wordEnd.isLineEnd()) {
+      if (wordEnd.getRight().isLineEnd()) {
         return;
       }
       start = start.getWordRight();
-      wordEnd = start.getCurrentWordEnd();
+      wordEnd = start.getCurrentWordEnd(true);
```

Comment:
Gah - we need to use more keyword arguments. e.g. i have no idea what this `true` means, but if it was `start.getCurrentWordEnd({ exclusive: true })`, that would be much better.

Not pertinent to this PR, but it's been on my mind.